src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Import/ImportFormHandler.php

```diff
@@ -26,56 +26,64 @@
 
 namespace PrestaShopBundle\Form\Admin\Configure\AdvancedParameters\Import;
 
-use PrestaShop\PrestaShop\Core\Form\FormDataProviderInterface;
-use PrestaShop\PrestaShop\Core\Form\FormHandlerInterface;
 use PrestaShop\PrestaShop\Core\Hook\HookDispatcherInterface;
+use PrestaShop\PrestaShop\Core\Import\Configuration\ImportConfigInterface;
 use Symfony\Component\Form\FormBuilderInterface;
 
 /**
- * Class ImportDataConfigurationFormHandler defines a form handler for import data configuration form.
+ * Class ImportFormHandler defines a form handler of import forms.
  */
-final class ImportDataConfigurationFormHandler implements FormHandlerInterface
+class ImportFormHandler implements ImportFormHandlerInterface
```

Comment:
We might merge this with a FormHandler for CRUD pages later, but it does not exist yet. Let's keep it in mind.